Makefile

```diff
@@ -13,3 +13,15 @@ test:
 
 vet:
 	go vet ./...
+
+# Docker commands
+.PHONY: docker_build docket_run
```

Comment:
For cleanliness, can we put the phony targets under each target? e.g.:

```
docker_build:
    docker build -t "$(IMAGE_NAME):latest" -t "$(IMAGE_NAME):$(GIT_COMMITSHA)" .
.PHONY: docker_builder
```

All the existing targets in here should have `.PHONY` as well (I'll add those later), and this will be cleaner for when it's a widely used convention.